src/commands/veloce/debug/logs.ts

```diff
@@ -56,15 +56,19 @@ export default class Org extends SfdxCommand {
     }
     const backendUrl: string | undefined = debugSession.backendUrl
 
-    try {
-      await axios.post(`${backendUrl}/services/dev-override/logs`, {}, {
-        headers
-      })
-    } catch ({ data }) {
-      this.ux.log(`Failed to list debug sessions: ${data as string}`)
-      return {}
+    this.ux.log('Following the backend logs...')
+
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
+      try {
+        const response = await axios.get(`${backendUrl}/services/dev-override/logs`, {headers});
+        if (response.data !== '') {
+          this.ux.log(response.data)
+        }
+      } catch (e: any) {
+        this.ux.log(`Failed to get logs: ${e as string}`)
```

Comment:
tut nuzhno toString() ?
as nuzhno kak mozhno menishe ispolizovatj

Reply:
this was moved just to have "better start" not as a source if ideal code. we are trying to write better code here